routes/experiments.js

```diff
@@ -1,7 +1,32 @@
 const experimentsRouter = require('express').Router();
 const ExperimentModel = require('../models/ExperimentModel');
 
-experimentsRouter.post('/', async (req, res) => {
+experimentsRouter.get('/all', async (req, res) => {
```

Comment:
Pourquoi /all ??
https://restfulapi.net/resource-naming/
```suggestion
experimentsRouter.get('/', async (req, res) => {
```

Reply:
changements fait